evelyn_chavez/python e/bike.py

```diff
@@ -0,0 +1,33 @@
+class Bike:
+    def __init__(self, price, max, miles):
+        self.price = price 
+        self.max = max
+        self.miles = 0
+
+    def displayinfo(self):
+        print('the bike price is '+str(self.price))
```

Comment:
since you are passing string while creating objects, //eg bike1 = Bike('$300', '60mph', '100miles')
it is not required to typecast self.price and self.max into str //str(self.price)